octopus-agent/src/main/resources/META-INF/build-agent-plugin-Octopus.TeamCity.xml

```diff
@@ -28,4 +28,5 @@
     <bean class="octopus.teamcity.agent.OctopusPromoteReleaseRunner"/>
     <bean class="octopus.teamcity.agent.OctopusPushPackageRunner"/>
     <bean class="octopus.teamcity.agent.OctopusPackPackageRunner"/>
+    <bean class="octopus.teamcity.agent.generic.OctopusGenericRunner"/>
```

Comment:
Would not registering the generic runner hide vNext build steps as per OctopusGenericRunner message "Octopus Generic Runner is not yet ready for use, please use existing steps"?

Reply:
So - the build-step will always fail validation (as we raise an invalidProperty when validating  - so technically, we probably don't need to have any agent stuff at all if we didn't want to